ScoutSuite/providers/aws/rules/findings/elbv2-no-access-logs.json

```diff
@@ -6,6 +6,7 @@
     "display_path": "elbv2.regions.id.vpcs.id.lbs.id",
     "conditions": [ "and",
         [ "Key", "equal", "access_logs.s3.enabled" ],
-        [ "Value", "equal", "false"]
+        [ "Value", "equal", "false"],
+        ["elbv2.regions.id.vpcs.id.lbs._GET_VALUE_AT_(elbv2.regions.id.vpcs.id.lbs.id).listeners.__values__.__mapParam__Protocol", "containAtLeastOneOf", ["TLS", "HTTP", "HTTPS"]]
```

Comment:
Shouldn't it just be `["TLS", "HTTPS"]`?

Reply:
Issue 617 is related to network load balancers. In the documentation of access logging for application load balancers (https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html) there is no mention that only the HTTPS listeners are getting logged, so I assumed that both of them (HTTP and HTTPS) will be logged, so the issue should be applicable to them.